src/screen-components/nearby-stop-places/NearbyStopPlacesScreenComponent.tsx

```diff
@@ -156,61 +158,61 @@ export const NearbyStopPlacesScreenComponent = ({
   const isFocused = useIsFocusedAndActive();
 
   return (
-    <FullScreenView
-      refreshControl={
-        // Quick fix for iOS to fix stuck spinner by removing the RefreshControl when not focused
-        isFocused || Platform.OS === 'android' ? (
-          <RefreshControl
-            refreshing={Platform.OS === 'ios' ? false : isLoading}
-            onRefresh={refresh}
-          />
-        ) : undefined
-      }
-      headerProps={headerProps}
-      parallaxContent={() => (
-        <Header
-          fromLocation={location}
-          updatingLocation={updatingLocation}
-          openLocationSearch={openLocationSearch}
-          setCurrentLocationOrRequest={setCurrentLocationOrRequest}
-          setLocation={(location: Location) => {
-            location.resultType === 'search' && location.layer === 'venue'
-              ? onSelectStopPlace(location)
-              : onUpdateLocation(location);
-          }}
-          mode={mode}
-          onAddFavoritePlace={onAddFavoritePlace}
-        />
-      )}
-    >
-      <ScreenReaderAnnouncement message={loadAnnouncement} />
-      {locationIsAvailable || !!location ? (
-        <StopPlaces
-          headerText={getListDescription()}
-          stopPlaces={orderedStopPlaces}
-          navigateToPlace={onSelectStopPlace}
-          testID="nearbyStopsContainerView"
-          location={location}
-          isLoading={isLoading}
-        />
-      ) : (
-        <EmptyState
-          title={t(NearbyTexts.stateAnnouncements.noAccessToLocationTitle)}
-          details={t(NearbyTexts.stateAnnouncements.noAccessToLocation)}
-          illustrationComponent={
-            <ThemedOnBehalfOf
-              height={90}
-              style={styles.emptyStopPlacesIllustration}
+    <>
```

Comment:
Why did this need to change? Do you have a screenshot from before vs after (though probably identical)?

Reply:
Right, I should have explained that in the PR 😅 The problem was the parallax scroll header, which hides the title until you scroll for a bit.  This change disables the parallax scroll on departures, making both the header and search fields always visible.